### PR TITLE
doc: add minutes for meeting 17 Sep 2020

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: meetings/2020-09-17.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+* We spent most of the time discussion the steps toward technical priorities.
+  * Discussion around how we get feedback from the right people get the right people involved
+  * Discussion on constituencies and what is important to them as next step for defining
+    Technical priorities. We should be able to map the priorities back to the constituencies.
+    * Noticed gap in that we are missing enterprises as a constituency
+    * We brainstormed the following addigion
+        * Organizations with investments in Node.js (ex: Enterprises, Government bodies, startups, non-profits) 
+    * Darcy will PR in.
+  * Discussed that ext focus area should be what does each constituencies need
```
Comment (mhdawson) [inline meetings/2020-09-17.md:43]:
```suggestion
* Discussed that next focus area should be what does each constituencies need
 
```

File: meetings/2020-09-17.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Node.js  Next 10 Years team Meeting 2020-09-17
+
+## Links
+
+* **Recording**:  https://youtu.be/xeXjCK4fCIA
```
Comment (trivikr) [inline meetings/2020-09-17.md:5]: nit
```suggestion
* **Recording**: https://youtu.be/xeXjCK4fCIA
```

File: meetings/2020-09-17.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+* Next 10 years for collaborators [#16](https://github.com/nodejs/next-10/issues/16)
+  * added by Mary, one of the things we should consider as next step  
+
+* Prioritizing Technical Values [#5](https://github.com/nodejs/next-10/issues/5)
+  * Values/directions document for core [#2](https://github.com/nodejs/next-10/issues/2)
+    * PR open don’t need more discussion https://github.com/nodejs/node/pull/35145
+
+* Discussion on what our next major focus area should be.  
+  * what we should do on the collaborator front [#16]
```
Comment (trivikr) [inline meetings/2020-09-17.md:32]:
```suggestion
  * what we should do on the collaborator front [#16](https://github.com/nodejs/next-10/issues/16)
```

File: meetings/2020-09-17.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  * steps toward technical priorities
+
+* We spent most of the time discussion the steps toward technical priorities.
+  * Discussion around how we get feedback from the right people get the right people involved
+  * Discussion on constituencies and what is important to them as next step for defining
+    Technical priorities. We should be able to map the priorities back to the constituencies.
+    * Noticed gap in that we are missing enterprises as a constituency
+    * We brainstormed the following addigion
```
Comment (trivikr) [inline meetings/2020-09-17.md:40]: addition?
```suggestion
    * We brainstormed the following addition
```

File: meetings/2020-09-17.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        * Organizations with investments in Node.js (ex: Enterprises, Government bodies, startups, non-profits)
```
Comment (trivikr) [inline meetings/2020-09-17.md:41]:
```suggestion
        * Organizations with investments in Node.js (eg: Enterprises, Government bodies, startups, non-profits) 
```

File: meetings/2020-09-17.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+* We spent most of the time discussion the steps toward technical priorities.
+  * Discussion around how we get feedback from the right people get the right people involved
+  * Discussion on constituencies and what is important to them as next step for defining
+    Technical priorities. We should be able to map the priorities back to the constituencies.
+    * Noticed gap in that we are missing enterprises as a constituency
+    * We brainstormed the following addigion
+        * Organizations with investments in Node.js (ex: Enterprises, Government bodies, startups, non-profits) 
+    * Darcy will PR in.
+* Discussed that next focus area should be what does each constituencies need
+
```
Comment (trivikr) [inline meetings/2020-09-17.md:44]:
```suggestion
```

File: meetings/2020-09-17.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    * Noticed gap in that we are missing enterprises as a constituency
+    * We brainstormed the following addigion
+        * Organizations with investments in Node.js (ex: Enterprises, Government bodies, startups, non-profits) 
+    * Darcy will PR in.
+* Discussed that next focus area should be what does each constituencies need
+ 
+    * This should actually include #16 as the Node.js maintainers are one of the constituencies
+       listed.
+    * Next meeting to brainstorm on what each constituency needs
+    * Action -> Darcy to setup boards, for each constituency , with following headings:need, don’t
```
Comment (trivikr) [inline meetings/2020-09-17.md:48]:
```suggestion
    * Action -> Darcy to set up boards, for each constituency, with following headings: need, don’t
```

File: meetings/2020-09-17.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+  * Discussion on constituencies and what is important to them as next step for defining
+    Technical priorities. We should be able to map the priorities back to the constituencies.
+    * Noticed gap in that we are missing enterprises as a constituency
+    * We brainstormed the following addigion
+        * Organizations with investments in Node.js (ex: Enterprises, Government bodies, startups, non-profits) 
+    * Darcy will PR in.
+* Discussed that next focus area should be what does each constituencies need
+ 
+    * This should actually include #16 as the Node.js maintainers are one of the constituencies
+       listed.
```
Comment (trivikr) [inline meetings/2020-09-17.md:46]:
```suggestion
    * This should actually include [#16](https://github.com/nodejs/next-10/issues/16) as the Node.js maintainers
    are one of the constituencies listed.
```

**Comment by mhdawson** [comment]
@nodejs/next-10 could use a few reviewers so this can land.

**Review by trivikr — APPROVED** [review]

**Pull request merged**